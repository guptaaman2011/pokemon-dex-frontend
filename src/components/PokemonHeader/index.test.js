import React from "react";
import { shallow } from "enzyme";
import { shallowToJson } from "enzyme-to-json";
import { PokemonHeader } from "..";

describe("PokemonHeader", () => {
  it("should render without breaking", () => {
    const wrapper = shallow(<PokemonHeader />);
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
