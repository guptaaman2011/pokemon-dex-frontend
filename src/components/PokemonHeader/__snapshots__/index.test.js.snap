// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PokemonHeader should render without breaking 1`] = `
<Layout
  className="layout"
>
  <Header
    className="header"
  >
    <div
      className="logo"
    >
      <img
        alt="PokemonImage"
        height="40px"
        src="/pokeapi_256.png"
      />
    </div>
  </Header>
</Layout>
`;
